x-pack/plugins/alerts/server/task_runner/task_runner.test.ts

```diff
@@ -206,9 +210,138 @@ describe('Task Runner', () => {
       references: [],
     });
     await taskRunner.run();
-    expect(taskRunnerFactoryInitializerParams.actionsPlugin.execute).toHaveBeenCalledTimes(1);
-    expect(taskRunnerFactoryInitializerParams.actionsPlugin.execute.mock.calls[0])
-      .toMatchInlineSnapshot(`
+    expect(actionsClient.enqueueExecution).toHaveBeenCalledTimes(1);
+    expect(actionsClient.enqueueExecution.mock.calls[0]).toMatchInlineSnapshot(`
+                  Array [
+                    Object {
+                      "apiKey": "MTIzOmFiYw==",
+                      "id": "1",
+                      "params": Object {
+                        "foo": true,
+                      },
+                      "spaceId": undefined,
+                    },
+                  ]
+          `);
+
+    const eventLogger = taskRunnerFactoryInitializerParams.eventLogger;
+    expect(eventLogger.logEvent).toHaveBeenCalledTimes(3);
+    expect(eventLogger.logEvent.mock.calls).toMatchInlineSnapshot(`
```

Comment:
nit: I would maybe assert on more specifics here or do multiple assertions. I've caught myself updating large snapshots without catching a change that shouldn't of been there.